src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-list/pool-list.component.ts

```diff
@@ -146,7 +146,7 @@ export class PoolListComponent implements OnInit {
       },
       {
         prop: 'last_change',
-        name: this.i18n('Last Change'),
+        name: this.i18n('Latest Edit'),
```

Comment:
this will not fix the issue for other locales if their text is larger than the `English` one.